### PR TITLE
Add STEG Electronics

### PR DESCRIPTION
**Pull request by jokay**
Adds [STEG Electronics](https://www.steg-electronics.ch).

rank-steg

### CODE REVIEW

**Comment by jokay** [comment]
@phallobst any idea why this one fails?

File: _data/retail.yml [hunk 1]
```diff
@@ -416,6 +416,16 @@ websites:
     facebook: staples
     img: staples.png
 
+  - name: STEG Electronics
+    url: https://www.steg-electronics.ch
+    img: steg.png
+    twitter: stegelectronics
```
Comment (Carlgo11) [inline _data/retail.yml]: Like the error message said:
```suggestion
    twitter: STEGelectronics
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/retail.yml (hunk 1 above)
Comment (jokay) [reply _data/retail.yml]: Thank you, I have no access to the [workflow tests](https://circleci.com/gh/2factorauth/twofactorauth/1770?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link) so I can't see the error message.

Comment (Carlgo11) [reply _data/retail.yml]: It doesn't allow you to see the logs even when you've logged in? Odd 🤔

**Review by Carlgo11 — APPROVED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**